decidim-core/app/jobs/decidim/hide_child_resources_job.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Decidim
+  class HideChildResourcesJob < ApplicationJob
+    queue_as :user_report
+
+    def perform(resource, user_id)
+      spam_user = resource.organization.users.find_or_initialize_by(email: Decidim::Ai::SpamDetection.reporting_user_email)
```

Comment:
We're adding a dependency to the optional module `decidim-ai`

Reply:
done in [51081d2](https://github.com/decidim/decidim/pull/13554/commits/51081d23a2980e9f331207d52aa0f8f7071fc563)